src/extensions/default/HTMLCodeHints/main.js

```diff
@@ -319,10 +318,9 @@ define(function (require, exports, module) {
                 
                 // If we're at an attribute value, check if it's an attribute name that has hintable values.
                 if (this.tagInfo.attr.name) {
-                    var hintsAndSortFunc = this._getValueHintsForAttr({queryStr: query},
-                                                                      this.tagInfo.tagName,
-                                                                      this.tagInfo.attr.name);
-                    var hints = hintsAndSortFunc.hints;
+                    var hints = this._getValueHintsForAttr({queryStr: query},
```

Comment:
@humphd please ignore this comment. I thought this was a multiline statement as that is what my browser showed me. And here I am back at home I see that it is just one line change.